fixes/logr.go

```diff
@@ -339,16 +390,8 @@ func (f *logrFix) fixError(selexpr *ast.SelectorExpr, callexpr *ast.CallExpr) {
 	for i, arg := range callexpr.Args {
 		t := f.loader.TypeInfo().Types[arg].Type
 		f.log.V(5).Info("arg", "idx", i, "type", t.String())
-		if t.String() == "error" {
```

Comment:
I think it might still be valuable to check this.  If I have 2 arguments, and one is `error` type and the other implements `error`, I'd prefer to choose the typed one.

Reply:
Might also be worth checking what golang.org/x/exp/errors/fmt does (the go 2 errors prototype).  I think it uses the last one, but I only glanced at the source code.

Reply:
(agreed on that point though)